code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -605,7 +605,11 @@ GLOBAL_VAR(station_nuke_source)
 
 /proc/KillEveryoneOnStation()
 	for(var/mob/living/victim as anything in GLOB.mob_living_list)
-		if(victim.stat != DEAD && is_station_level(victim.z))
+		var/turf/M = get_turf(victim)
+		if(prob (10) && istype(victim.loc, /obj/structure/closet/secure_closet/freezer/))
```

Comment:
dont use spaces for proc args, and don't put slashes at the end of typepaths

Reply:
also wouldnt a reverse order be better? check if someone is in a freezer and then a 10% chance

Reply:
Sorry about my bad coding, I will get on it soon.

Reply:
Wait no I got the meaning of proc args wrong.

Reply:
Sorry but I gonna need some time to solve the proc args problem. As it is pretty late for me now, I will handle them tomorrow.

Reply:
Wait no, oh god. Were you talking about the space in prob, I just notice it.  That, I can fix immediately.
Or you were indeed talking about proc args being empty.

Reply:
yeah its good now